src/Composer/InstallFilesPlugin.php

```diff
@@ -38,16 +38,20 @@ class InstallFilesPlugin implements EventSubscriberInterface, PluginInterface
 
     public function activate(Composer $composer, IOInterface $io): void
     {
+        $io->write('<info>[Qualitou] Activation</info>');
+
         $this->composer = $composer;
         $this->io = $io;
     }
 
     public function deactivate(Composer $composer, IOInterface $io): void
     {
+        $io->write('<info>[Qualitou] Désactivation</info>');
     }
 
     public function uninstall(Composer $composer, IOInterface $io): void
     {
+        $io->write('<info>[Qualitou] Désinstallation</info>');
```

Comment:
```suggestion
        $io->write('<info>[Qualytou] Désinstallation</info>');
```

Reply:
Y a un problème avec le nom de ce bundle je trouve.

C'est une contraction de **qual**it**y to**ol ? Si oui même en anglais il y a un 'i' et ça devrait être too au lieu de tou

Sinon vu que les commentaires et README sont en français (pour un bundle public à vocation grand public c'est assez étrange, quand même, si c'est un bundle bien bleu blanc rouge faudrait l'appeller je ne sais pas, qualoutils ?

Reply:
Débat pas lié a la PR a priori ;)